Documentation/ceph-object-store-crd.md

```diff
@@ -89,7 +89,7 @@ The gateway settings correspond to the RGW daemon settings.
 
 * `type`: `S3` is supported
 * `sslCertificateRef`: If the certificate is not specified, SSL will not be configured. If specified, this is the name of the Kubernetes secret that contains the SSL certificate to be used for secure connections to the object store. Rook will look in the secret provided at the `cert` key name. The value of the `cert` key must be in the format expected by the [RGW service](https://docs.ceph.com/docs/master/install/ceph-deploy/install-ceph-gateway/#using-ssl-with-civetweb): "The server key, server certificate, and any other CA or intermediate certificates be supplied in one file. Each of these items must be in pem form."
-* `port`: The port on which the RGW pods and the RGW service will be listening (not encrypted).
+* `port`: The port on which the RGW pods and the RGW service will be listening (not encrypted). This will be internal port for rgw daemon only if host network is enabled. Otherwise it uses 8080 by default.
```

Comment:
```suggestion
* `port`: The port on which the RGW pods and the RGW service will be listening (not encrypted). This will be an internal port for the RGW daemon only if host network is enabled. Otherwise it uses 8080 by default.
```

Reply:
Is 8080 an internal rgw default, or it is just referring to the default in rgw-external.yaml?

Reply:
@travisn it is more like set by rook, 
```go
   //pkg/operator/ceph/object/config.go
    rgwPortInternalPort int32 = 8080 

//pkg/operator/ceph/object/spec.go
func (c *clusterConfig) generateLiveProbePort() intstr.IntOrString {
    // The port the liveness probe needs to probe
    // Assume we run on SDN by default
    port := intstr.FromInt(int(rgwPortInternalPort))

    // If Host Networking is enabled, the port from the spec must be reflected
    if c.clusterSpec.Network.IsHost() {
        if c.store.Spec.Gateway.Port == 0 && c.store.Spec.Gateway.SecurePort != 0 && c.store.Spec.Gateway.SSLCertificateRef != "" {
            port = intstr.FromInt(int(c.store.Spec.Gateway.SecurePort))
        } else {
            port = intstr.FromInt(int(c.store.Spec.Gateway.Port))
        }
    }

    return port
}
```